sal-common/src/main/java/org/ow2/proactive/sal/model/EdgeNode.java

```diff
@@ -29,54 +29,83 @@
 @Setter
 @Table(name = "EDGE_NODE")
 public class EdgeNode extends AbstractNode {
+    public static final String ANY_JOB_ID = "any";
+
+    // Constants for JSON properties
+    public static final String JSON_NAME = "name";
+
+    public static final String JSON_LOGIN_CREDENTIAL = "loginCredential";
+
+    public static final String JSON_IP_ADDRESSES = "ipAddresses";
+
+    public static final String JSON_NODE_PROPERTIES = "nodeProperties";
+
+    public static final String JSON_PORT = "port";
+
+    public static final String JSON_REASON = "reason";
+
+    public static final String JSON_DIAGNOSTIC = "diagnostic";
+
+    public static final String JSON_USER_ID = "userId";
+
+    public static final String JSON_ALLOCATED = "allocated";
+
+    // edge jobID corresponds to the ProActive job name
+    public static final String JSON_JOB_ID = "jobId";
+
+    public static final String JSON_SYSTEM_ARCH = "systemArch";
+
+    public static final String JSON_SCRIPT_URL = "scriptURL";
+
+    public static final String JSON_JAR_URL = "jarURL";
```

Comment:
I think all these would be better in the EdgeDefinition class.
Also remove the comment `// Constants for JSON properties`

Reply:
you are right :) done